icons/codeberg.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Codeberg icon</title><path d="M11.955.49C5.345.515 0 5.88 0 12.49a12 12 0 0 0 1.832 6.373c3.365-4.37 6.764-8.713 10.12-13.09a.25.25 0 0 1 .216-.1.25.25 0 0 1 .18.098l9.959 12.866A12 12 0 0 0 24 12.49c0-6.627-5.373-12-12-12h-.045zm.213 5.865l4.578 17.155a12 12 0 0 0 5.277-4.424z"/></svg>
```

Comment:
Looks good to me, though something about the top of the stroke doesn't sit right.
![image](https://user-images.githubusercontent.com/1830867/104458481-5d671900-55a3-11eb-8b0b-3e510eb6c181.png)

Might just be that the stroke is pushing just the right side of the pyramid across. Could another @simple-icons/maintainers take a look at this and add their feedback?

Reply:
I agree with @adamrusted that the cut-out shouldn't encroach on the circle. Something more along these lines maybe but retaining, if possible, the rounding on the tip of the triangle.

![](https://user-images.githubusercontent.com/15157491/108499848-d7777580-72a6-11eb-84e8-32b33eec9c1c.png)

Reply:
Fixed.

It has been quite a while since I've worked with SVGs (and contributed to Simple Icons), but hopefully this is ready to be merged now. 😊

Reply:
Good to have you back @fbernhart 🤣
I'll take a look at this one shortly and hopefully merge it ahead of the weekend release.